transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditor.java

```diff
@@ -349,8 +356,38 @@ public void init(IEditorSite site,
 
         IFile configFile = ((FileEditorInput)getEditorInput()).getFile();
         IJavaProject javaProject = JavaCore.create(configFile.getProject());
-        try {
-            loader = (URLClassLoader)JavaUtil.getProjectClassLoader(javaProject, getClass().getClassLoader());
+        IProgressMonitor monitor = new NullProgressMonitor();
+        try {loader = (URLClassLoader)JavaUtil.getProjectClassLoader(javaProject, getClass().getClassLoader());
```

Comment:
what do you think to extract the code responsible to do the migration in a specific method (or even class)? it will improve readability by avoiding a method of 74 lines and will more easily allow to reuse thee code for improving the "Change camel version"
We can also imagine more easily a unit test checking the replacement of the values in the file.